testbench/rest_server.py

```diff
@@ -879,7 +879,7 @@ def download_object_get(bucket_name, object_name):
 def object_insert(bucket_name):
     db.insert_test_bucket()
     bucket = db.get_bucket(bucket_name, None).metadata
-    upload_type = flask.request.args.get("uploadType")
+    upload_type = flask.request.args.get("uploadType", "resumable")
```

Comment:
If the Ruby library depends on this, it is depending on undocumented behavior.  The `uploadType` parameter is a **required** parameter:

https://cloud.google.com/storage/docs/json_api/v1/objects/insert#parameters

Reply:
I tried this as well... interestingly the JSON API seems to accept requests without UploadType, but it defaults to a simple upload rather than resumable.